src/Screens/Onboarding/SelectTopic.tsx

```diff
@@ -45,7 +45,7 @@ export default ({ navigation }: Props) => {
 
   return (
     <OnboardingBackground>
-      <Card style={styles.card}>
+      <Card style={styles.card} testID="selectTopicCard">
```

Comment:
I have never used testIds with any project. Should we have some common place where we import these names and we could also type them on the go like MessageId type implementation with translation